test/test_distributions.py

```diff
@@ -1279,6 +1327,23 @@ def test_categorical_log_prob_with_logits(self):
             log_pdf_prob_0 = categorical.log_prob(Variable(tensor_type([1, 0])))
             self.assertEqual(log_pdf_prob_0.data[0], -float('inf'), allow_inf=True)
 
+    def test_multinomial_log_prob(self):
```

Comment:
This fails, but it can fixed with increasing the resolution of `_get_clamping_buffer`

Reply:
Is this why you need the `.machine_epsilon` or something similar?

Reply:
Yes